Suggest customers to upvote existing issues rather than file a new one (#32070)
dotnet/aspnetcore: CONTRIBUTING.md

Start a discussion on the [repository issue tracker](https://github.com/dotnet/aspnetcore/issues).

## Bugs and feature requests?

> **IMPORTANT: If you want to report a security related issue, please see the `Reporting security issues and bugs` section below.**

Reviewer: Consider putting a ❗ icon before the "IMPORTANT".

> Before reporting a new issue, try to find an existing issue if one already exists. If it already exists, upvote (👍) it. Also consider adding a comment with your unique scenarios and requirements related to that issue.  Upvotes and clear details on the issue's impact help us prioritize the most important issues to be worked on sooner rather than later.

Reviewer: I
```suggestion
Before reporting a new issue, try to find an existing issue if one already exists. If it already exists, upvote (👍) it. Also consider adding a comment with your unique scenarios and requirements related to that issue.  Upvotes and clear details on the issue's impact help us prioritize the most important issues to be worked on sooner rather than later. If you can't find one, that's okay, we'd rather a duplicate report than none.
```

Reviewer: We should be careful not to scare people away by having language that's too strong.

> Before reporting a non-security related bug or filing a new issue try to find an existing issue if such exists. If found, upvote (👍) it and add a comment about your scenario to that issue. Our team pays attention to reactions a lot and having an issue with many upvotes or other positive reactions will help us prioritize it to be worked on sooner rather than later.

Author: ```suggestion
Before reporting a non-security related bug or filing a new issue try to find an existing issue if such exists. If found, upvote (👍) it and add a comment about your unique scenario to that issue. Our team pays attention to reactions a lot and having an issue with many upvotes or other positive reactions will help us prioritize it to be worked on sooner rather than later.
```

> Before reporting a non-security related bug or filing a new issue try to find an existing issue if such exists. If found, upvote (👍) it and add a comment about your scenarioto to that issue. Our team pays attention to reactions a lot and having an issue with many upvotes or other positive reactions will help us prioritize it to be worked on sooner rather than later.

Reviewer 2: ```suggestion
Before reporting a non-security related bug or filing a new issue try to find an existing issue if such exists. If found, upvote (👍) it and add a comment about your scenario to that issue. Our team pays attention to reactions a lot and having an issue with many upvotes or other positive reactions will help us prioritize it to be worked on sooner rather than later.
```

Reviewer 2: Maybe it's just me, but I'm not a fan of people commenting the same thing over and over on an issue, and I definitely don't want to encourage people to do that.

Maybe you can reword this to say something like "add a comment about any unique scenarios you have".

> Before reporting a new issue, try to find an existing issue if one already exists. If it already exists, upvote (👍) it. Also consider adding a comment with your unique scenarios and requirements related to that issue.  Upvotes and clear details on the issue's impact help us prioritize the most important issues to be worked on sooner rather than later. If you can't find one, that's okay, we'd rather a duplicate report than none.

Author: ```suggestion
Before reporting a new issue, try to find an existing issue if one already exists. If it already exists, upvote (👍) it. Also consider adding a comment with your unique scenarios and requirements related to that issue.  Upvotes and clear details on the issue's impact help us prioritize the most important issues to be worked on sooner rather than later. If you can't find one, that's okay, we'd rather have a duplicate report than none.
```

## Bugs and feature requests?
**IMPORTANT: If you want to report a security related issue, please see the `Reporting security issues and bugs` section below.**

> Before reporting a non-security related bug or filing a new issue try to find an existing issue if such exists. If found, upvote (👍) it and add a comment about your unique scenario to that issue. Our team pays attention to reactions a lot and having an issue with many upvotes or other positive reactions will help us prioritize it to be worked on sooner rather than later.

Reviewer 3: ```suggestion
Before reporting a new issue, try to find an existing issue if one already exists. If it already exists, upvote (👍) it. Also consider adding a comment with your unique scenarios and requirements related to that issue.  Upvotes and clear details on the issue's impact helps us prioritize the most important issues to be worked on sooner rather than later.
```
